config/locales/en.yml

```diff
@@ -246,6 +246,7 @@ en:
         new_layout: "New layout"
         description_roadmap: "Sharetribe is currently in the process of rolling out a new and improved layout. During the transition period, you can select which parts of your marketplace should continue using the old layout using the menu below. New features will only be available on the new layout."
         description_process: "Support for the old layout will be removed in the future. The Sharetribe team will notify you before this happens. Questions? Contact the team via %{support_link}."
```

Comment:
Now that `description_process` and `description_roadmap` aren't used anymore, they can and should be removed from the `en.yml` file.